### PR TITLE
Fp 2607 bad styles depending on dependency tree lib react problem detected in fleet

### PR DESCRIPTION
**Pull request by quirinpa**
- [FP-2607](https://movai.atlassian.net/browse/FP-2607): Bad styles depending on dependency tree (lib-react problem) - detected in fleet

[FP-2607]: https://movai.atlassian.net/browse/FP-2607?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/Components/ProfileMenu/ProfileMenu.js
Comment (manuelnogueiramov) [inline src/Components/ProfileMenu/ProfileMenu.js:1]: What the hell is this file!?

**Review by manuelnogueiramov — CHANGES_REQUESTED** [review]
I don't like the ammount of importants we are adding in this PR.
Also, I don't understand why you added a weird ProfileMenu file.
Keep in mind that we want to remove types altogether from the codebase, I saw some mention of types in this PR. Adding MORE types will not help us remove it. So, let me know what happened here.

File: src/Components/ProfileMenu/ProfileMenu.js
Comment (quirinpa) [reply src/Components/ProfileMenu/ProfileMenu.js:1]: I did not add a dependency on @tty-pt/styles to lib-react.
If I did, it was a mistake.

Comment (quirinpa) [reply src/Components/ProfileMenu/ProfileMenu.js:1]: That's a compiled file. Has nothing to do with @tty-pt/styles

Comment (quirinpa) [reply src/Components/ProfileMenu/ProfileMenu.js:1]: Nope. I didn't.

That's a compiled file. It was added by mistake. I'll remove it soon.

Comment (quirinpa) [reply src/Components/ProfileMenu/ProfileMenu.js:1]: You are wrong about what you're saying. I didn't add types as a dependency.

This file was added by mistake and it does not reference @tty-pt/styles.

**Comment by quirinpa** [comment]
We did not add types. That is incorrect. Nor styles.

**Review by manuelnogueiramov — APPROVED** [review]

**Pull request merged**